.github/workflows/test-create-pr-workflow.yml

```diff
@@ -0,0 +1,27 @@
+name: Test Create PR workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      target_branch:
+        description: 'Target branch for the PR (main or test)'
+        required: true
+        default: 'main'
+      tag_name:
+        description: 'Tag name for release (optional)'
+        required: false
+        default: ''
+      prerelease:
+        description: 'Is this a prerelease? (true/false)'
+        type: boolean
+        required: false
+        default: false
+
+jobs:
+  test-create-pr-to-main-workflow:
+    name: Test create PR workflow
+    uses: ./.github/workflows/create-pr.yml
+    with:
+      target_branch: ${{ github.event.inputs.target_branch }}
+      tag_name: ${{ github.event.inputs.tag_name }}
+      prerelease: ${{ github.event.inputs.prerelease == 'true' }}
```

Comment:
Will it work if we just pass `github.event.inputs.prerelease` here?

Reply:
No, type from the `workflow_dispatch` input casts to the string I don't know why.